sbnci/PlottingScripts/CompareDataDistributions.C

```diff
@@ -139,6 +139,19 @@ double getMax(TH1* hData, TH1* hMC) {
     else return datamax;
 }
 
+void getHistMaxMinWithError(TH1* hist, double &histMax, double &histMin) {
+  double maxi = -std::numeric_limits<double>::max();
+  double mini = std::numeric_limits<double>::max();
+
+  for(int i=1; i<hist->GetNbinsX()+1; ++i)
+    {
+      maxi = std::max(maxi, hist->GetBinContent(i) + hist->GetBinErrorUp(i));
+      mini = std::min(mini, hist->GetBinContent(i) + hist->GetBinErrorLow(i));
```

Comment:
If I understand this correctly, maybe for `mini` you want to subtract the lower error bar value?
`mini = std::min(mini, hist->GetBinContent(i) - hist->GetBinErrorLow(i));`

Reply:
I absolutely do, good catch!